packages/account/src/Components/poo-not-required/not-required.jsx

```diff
@@ -0,0 +1,16 @@
+import { Icon } from '@deriv/components';
```

Comment:
Please organise all new files for new feature in separate folder, like it is already done for poa and poi